src/game/game-store.js

```diff
@@ -2,4 +2,6 @@ import { createStore } from 'redux';
 import { gameReducer } from './reducers/game-reducer.js';
 
 const store = createStore(gameReducer);
+window.store = store;
```

Comment:
Just curious, maybe it makes sense to add NODE_END and add the store to window conditionally?